internal/server/iigointernal/executive.go

```diff
@@ -24,7 +24,7 @@ type executive struct {
 // loadClientPresident checks client pointer is good and if not panics
 func (e *executive) loadClientPresident(clientPresidentPointer roles.President) {
 	if clientPresidentPointer == nil {
-		panic(fmt.Sprintf("Client '%v' has loaded a nil president pointer", e.ID))
+		panic(fmt.Sprintf("Client '%v' has loaded a nil president pointer", e.PresidentID))
```

Comment:
Only change we made was rename ID to PresidentID. We did not implement the panic

Reply:
#168 author @Neelesh99 approver @mpardalos

Reply:
Hey soz just reading this now, the premise of this panic is that if any client for some reason is passing in a nil pointer for this, it is a fundamental problem maybe in the client's implementation. Since we have baseRoles there should be no scenario which has a nil pointer being passed from the client side.